### PR TITLE
Awesome Books: 

### PR DESCRIPTION
**Pull request by solorics**
### Refractor Using Classes 💯 

>. Use JavaScript classes instead of objects and arrays. ✅ 

>. Create class methods to add and remove books. ✅ 

>. Maintain all of the original functionality of the application ✅ 

>. used only plain JavaScript. ✅ 

>. No Lintere's Errors 👍

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @solorics & @Nedramrez, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have a professional readme. Make sure to add an updated readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Nedramrez** [comment]
I'm very sorry. I thought that we've already done it. Forgot that it was different branch.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+tr {
+  display: flex;
+  flex-direction: column;
+  font-size: 20px;
+}
+
+button {
+  padding: 3px 10px;
+  margin-bottom: 1em;
+}
```
Comment (AmaduKamara) [inline style.css]: Your project is refactored correctly and works fine as expected, but I think you're missing out on the styling.

- [ ] For this project, alongside the refactor, you're required to style your app to look similar to the wireframe given.

**See the requirements**

Screenshot 2023-01-13 at 9 22 34 AM

**Here is how your app looks**

Screenshot 2023-01-13 at 9 23 48 AM

- [x] Kindly work on your styles for your project to be approved 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️  

Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Refactored your app to use classes as expected 👍🏽
✅ Add and remove features work fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👏 You did well to make changes as requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nedramrez** [comment]
Thank you!

**Pull request merged**